Makefile

```diff
@@ -301,11 +308,18 @@ minikube-iso-%: # build minikube iso
                 mv $(BUILD_DIR)/buildroot/output-x86_64/images/rootfs.iso9660 $(BUILD_DIR)/minikube-amd64.iso; \
         fi;
 
-.PHONY: iso-prepare-%
+# iso-prepare runs on the host, it sets up the buildroot rootfs-overlay content
+
+.PHONY: iso-prepare-x86_64 iso-prepare-aarch64
 iso-prepare-%: buildroot
 	echo $(VERSION_JSON) > deploy/iso/minikube-iso/board/minikube/$*/rootfs-overlay/version.json
 	echo $(ISO_VERSION) > deploy/iso/minikube-iso/board/minikube/$*/rootfs-overlay/etc/VERSION
 	cp deploy/iso/minikube-iso/arch/$*/Config.in.tmpl deploy/iso/minikube-iso/Config.in
+
+# iso-config runs in the builder, it creates the buildroot .config file to use
```

Comment:
maybe use iso-prepare-host and iso-prepare-container? The comment is very helpful but the name can explain the purpose of the target.

Why do need to separate the setup steps on the host and in the container?